docs/service_spec.md

```diff
@@ -291,6 +299,8 @@ A `POST` request indicates that the test harness wants to start an instance of t
   * `wrapper` (object, optional): If specified contains wrapper configuration.
     * `name`: The name of the wrapper.
     * `version`: The version of the wrapper.
+  * `proxy` (object, optional): If specified contains proxy configuration.
+    * `httpProxy` (string, optional): An HTTP proxy, of the form `http://host:port`.
```

Comment:
I feel like this should be scheme, host, and port as independent fields. Easier to form a URL than parse one.